### ISSUE TITLE
Support for directly reading ENV vars from dotenv files

### ISSUE THREAD

**Issue by praveen-nj** [boxblinkracer/cypress-testrail#8]
It'd be nice if the plugin can read ENV vars from `process.env` along with the currently supported ways.

Benefits,
1. Less boilerplate code during setup
2. Easy to run from CI

**Comment by boxblinkracer** [comment]
Hi there

thank you for this

you mean like from the Cypress.env.json only from the process.env?

**Comment by praveen-nj** [comment]
No, like directly from `process.env` without the need for having a `cypress.env.json` file

Idk if this will work but here's an example

ConfigServie.js --> getDomain()
```
if (process.env.TESTRAIL_DOMAIN !== undefined && process.env.TESTRAIL_DOMAIN !== '') {
  return process.env.TESTRAIL_DOMAIN
}
```

**Comment by boxblinkracer** [comment]
hi there
sorry for not coming back earlier,
that makes sense, ill add it to the roadmap :)

**Comment by boxblinkracer** [comment]
Hi

thank you for this great idea
process.env is supported in the next release
you can already try it with dev-master if you want

**Comment by boxblinkracer** [comment]
released with 2.10.0
